src/adapter/src/catalog/apply.rs

```diff
@@ -1023,6 +1027,12 @@ impl CatalogState {
                 &comment.comment,
                 diff,
             )],
+            StateUpdateKind::AuditLog(audit_log) => {
+                assert_eq!(diff, 1, "audit log is append only");
```

Comment:
~~What about migrations of the audit log? It's possible we could insert some retractions and I don't think they'd necessarily be consolidated away immediately?~~

Nvm, looks like the Catalog consolidates on read!